lib/dangermattic/plugins/android_misc_checks.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin for miscellaneous checks for Android.
+  class AndroidMiscChecks < Plugin
```

Comment:
I'm not a fan of this "fits-it-all bucket" kind of thing. This can easily become a place to dump everything blindly over time and make it a bit of a mess with lots of unrelated checks.

> They were both in the general iOS/Android rules file and didn't really fit into the https://github.com/Automattic/dangermattic/pull/9, but so far they're not really close to anything else, therefore miscellaneous checks. As new plugins added with time, possibly we can group them in another way if needed.

I think it'd make more sense to go the other way around, i.e. make the plugin have a name that relates to a specific area, like `AndroidStringsChecks`, that can evolve later to add more checks in that same area (I can already imagine checks for ensuring that a string with an existing key didn't have its value mutated (p1690456266010749/1690454415.738839-slack-CRF8CPYAK).
And if over time we realize that our grouping of the checks could have been done in a different way and split into different plugins, we could always refactor (and at that time we'll then have enough examples of plugins and checks in our toolset to reason about a better grouping if needed)

Reply:
> I think it'd make more sense to go the other way around, i.e. make the plugin have a name that relates to a specific area, like `AndroidStringsChecks`, that can evolve later to add more checks in that same area (I can already imagine checks for ensuring that a string with an existing key didn't have its value mutated)

I agree, it is too easy to have a "basement" plugin where you just throw stuff you don't know where else to put. I'll move things around to specific plugins for now. Good suggestion!